packages/api/src/apps/api/app.ts

```diff
@@ -45,6 +46,11 @@ export default async (env: ProcessEnv) => {
   // services can emit events when necessary, though for now any services that depend on events must be in same process
   const serviceEvents = new Events();
 
+  const empsActiveStore = stores.JsMap<string, tables.emps.Data>();
+  const empsExpiredStore = stores.JsMap<string, tables.emps.Data>();
+  const empsActiveTable = tables.emps.JsMap("Active Emp", empsActiveStore);
+  const empsExpiredTable = tables.emps.JsMap("Expired Emp", empsExpiredStore);
```

Comment:
can we change the table exports to not specify the store type. Might want to just consider exposing them like this

`tables.emps.Table("Expired Emp", empsExpiredStore)`  or `tables.Emps("Expired Emp", empsExpiredStore)` .

First option gives some better flexibility export things other than table, but the first one is a bit cleaner if we only expect to use table.  wdyt?

Also you may not explicitly need to define stores, maybe you did it here for testing, but since you have defaults you can probably just instance the tables, like with blocks/erc20s.